example/server/server_test.go

```diff
@@ -37,7 +38,7 @@ func TestMain(m *testing.M) {
 	}()
 	greeterClient = example.NewGreeterClient(greeterConn)
 
-	os.Exit(m.Run())
+	defer os.Exit(m.Run())
```

Comment:
Is this `defer` useful here? Considering it is already the last statement of the function.

Reply:
Right, we don't need `defer` here.